mods/mc_teacher/maps/MKRF512_dtm.conf

```
@@ -3,11 +3,14 @@ name = Malcolm Knapp Research Forest - digital terrain model
 spawn_pos_x = 279
 spawn_pos_y = 63
 spawn_pos_z = 161
-schematic_size_x = 513
+schematic_size_x = 512
 schematic_size_y = 175
-schematic_size_z = 513
-license = CC BY-SA 4.0
+schematic_size_z = 512
+offset_x = -1
+offset_y = -1

```

The bottom realm border seems to be clipping the lowest layer of blocks, which is most noticeable along the north edge of the realm
```suggestion
offset_y = 0
```